fhttp/http_client.go

```diff
@@ -396,6 +399,11 @@ func (c *Client) Fetch() (int, []byte, int) {
 	return code, data, 0
 }
 
+// GetIPAddress get the ip address that DNS resolves to when using stdClient.
+func (c *Client) GetIPAddress() string {
+	return <-stdClientIP
```

Comment:
any chance this isn't yet there when this is called? would block?

Reply:
removed